BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -28,18 +41,17 @@ void BSP_Timer_Start(void) {
  * @return  Number of ticks
  */
 uint32_t BSP_Timer_GetTicksElapsed(void) {
-    // TODO
-    return 42;
+    return clock() - t;
+    
 }
 
 /**
- * @brief   Gets the running frequency of the timer (time per tick)
+ * @brief   Gets the ticks/s from CLOCKS_PER_SEC
  * @param   None
- * @return  frequency in Hz
+ * @return  ticks/s
  */
 uint32_t BSP_Timer_GetRunFreq(void) {
-    // TODO
-    return 42;
+    return CLOCKS_PER_SEC;  //returns ticks/s instead because this function is only used internally
```

Comment:
Leave a `NOTE:` above the function call that explains why this function wasn't implemented. Also, just return 0. It is easier to tell it isn't used.